Sources/WebSocket.swift

```diff
@@ -135,6 +135,8 @@ open class FoundationStream : NSObject, WSStream, StreamDelegate  {
     private static let sharedWorkQueue = DispatchQueue(label: "com.vluxe.starscream.websocket", attributes: [])
     private var inputStream: InputStream?
     private var outputStream: OutputStream?
+    public var onCreateInputStream: ((InputStream)->Void)?
```

Comment:
If were performing surgery on this file already why not just add these to WSStream so we dont need to do casting below?